raydar/dashboard/server.py

```diff
@@ -107,6 +109,16 @@ async def update_table_rest(self, tablename: str, request: Request) -> Response:
         except BaseException as exception:
             raise HTTPException(503, f"Exception during data ingestion: {tablename} / {format_exc()}") from exception
 
+    @app.post("/save_workspace_layout")
+    async def save_workspace_layout(self, layout: dict) -> Response:
+        file_path = os.path.join(static_files_dir, "layouts", "default.json")
```

Comment:
this path is not expected to be writeable most of the time, workspace save/restore can be done in browser localstorage